src/CommunityToolkit.Aspire.Client.MassTransit.RabbitMQ/CommunityToolkit.Aspire.Client.MassTransit.RabbitMQ.csproj

```
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+      <AdditionalPackageTags>Client Aspire MassTransit RabbitMQ</AdditionalPackageTags>
+      <Description>A .NET Aspire client integration for MassTransit RabbitMQ.</Description>
+        <TargetFramework>net8.0</TargetFramework>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+    </PropertyGroup>
+
+
+  <ItemGroup>
+    <PackageReference Include="MassTransit.RabbitMQ" Version="8.3.2"/>
+    <PackageReference Include="OpenTelemetry.Extensions.Hosting" Version="1.10.0"/>
```

```suggestion
    <PackageReference Include="MassTransit.RabbitMQ" />
    <PackageReference Include="OpenTelemetry.Extensions.Hosting" />
```

NuGet CPM will handle versions